src/pod.rs

```diff
@@ -245,109 +257,126 @@ pub fn guarded_transmute_pod_many_pedantic<T: PodTransmutable>(bytes: &[u8]) ->
+pub unsafe fn guarded_transmute_pod_vec<T, G>(bytes: Vec<u8>) -> Result<Vec<T>, Error>
+where
+    T: PodTransmutable,
+    G: Guard,
+{
+    guarded_transmute_vec::<T, G>(bytes)
 }
 
-/// Trasform a byte vector into a vector of POD.
+/// Transform a byte vector into a vector of POD.
 ///
 /// The vector's allocated byte buffer will be reused when possible, and
 /// have as many instances of a type as will fit, rounded down.
 /// Extraneous data is ignored.
 ///
-/// # Errors
-///
-/// An error is raised in one of the following situations:
-///
-/// - The data does not have a memory alignment compatible with `T`. You will
-///   have to make a copy anyway, or modify how the data was originally made.
+/// # Safety
+/// 
+/// It is undefined behavior If the data does not have a memory alignment
+/// compatible with `T`. If this cannot be ensured, you will have to make a
+/// copy of the data, or change how it was originally made.
 ///
 /// # Examples
 ///
 /// ```no_run
-/// # use safe_transmute::guarded_transmute_pod_vec_permissive;
+/// # use safe_transmute::pod::guarded_transmute_pod_vec_permissive;
 /// # include!("../tests/test_util/le_to_native.rs");
 /// # fn main() {
 /// // Little-endian
+/// unsafe {
 /// # /*
-/// assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(vec![0x00, 0x01, 0x00, 0x02]),
+///     assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(vec![0x00, 0x01, 0x00, 0x02]),
 /// # */
-/// # assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(vec![0x00, 0x01, 0x00, 0x02].le_to_native::<u16>()),
-///            Ok(vec![0x0100, 0x0200]));
+/// #     assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(vec![0x00, 0x01, 0x00, 0x02].le_to_native::<u16>()),
+///                Ok(vec![0x0100, 0x0200]));
 /// # /*
-/// assert_eq!(guarded_transmute_pod_vec_permissive::<u32>(vec![0x04, 0x00, 0x00, 0x00, 0xED]),
+///     assert_eq!(guarded_transmute_pod_vec_permissive::<u32>(vec![0x04, 0x00, 0x00, 0x00, 0xED]),
 /// # */
-/// # assert_eq!(guarded_transmute_pod_vec_permissive::<u32>(vec![0x04, 0x00, 0x00, 0x00, 0xED].le_to_native::<u32>()),
-///            Ok(vec![0x0000_0004]));
-/// assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(vec![0xED]), Ok(vec![]));
+/// #     assert_eq!(guarded_transmute_pod_vec_permissive::<u32>(vec![0x04, 0x00, 0x00, 0x00, 0xED].le_to_native::<u32>()),
+///                Ok(vec![0x0000_0004]));
+///     assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(vec![0xED]), Ok(vec![]));
+/// }
 /// # }
 /// ```
 #[cfg(feature = "std")]
-pub fn guarded_transmute_pod_vec_permissive<T: PodTransmutable>(bytes: Vec<u8>) -> Result<Vec<T>, Error> {
-    check_alignment::<_, T>(&bytes)?;
-    unsafe { Ok(guarded_transmute_vec_permissive(bytes)) }
+#[deprecated(since = "0.11.0", note = "see `pod::guarded_transmute_pod_vec` for the equivalent behavior")]
+pub unsafe fn guarded_transmute_pod_vec_permissive<T: PodTransmutable>(bytes: Vec<u8>) -> Result<Vec<T>, Error> {
```

Comment:
Doesn't follow convention — has a full doc and isn't just a call to the funxion linked in `#[deprecated]`.
The `#[deprecated]` doesn't specify the Guard type, either.

Occurs on:
  *	L337
  *	L380